shard-operator-parent/shard-operator-v2/src/test/java/com/redhat/service/smartevents/shard/operator/v2/ManagedBridgeServiceImplTest.java

```diff
@@ -135,6 +138,20 @@ public void managedBridgeCreationTriggersController() {
                 });
     }
 
+    @Test
+    public void IsBridgeStatusChange() {
```

Comment:
@vaibhavjainwiz Can we have the opposite test please that verifies that if all conditions on a `ManagedBridge` are the same, then this method returns `false` as it's a key part of ensuring we only send updates to the Control Plane when we need to.

Reply:
done